odl/util/testutils.py

```diff
@@ -229,6 +229,11 @@ def is_subdict(subdict, dictionary):
         reason='Need --benchmark option to run'
     )
 
+    skip_if_no_contrib = pytest.mark.skipif(
```

Comment:
I guess we want this to be automatic for all tests in the folder and not something that users need to add themselves? Not sure if we can do that though